modules/ocf/manifests/repackage.pp

```diff
@@ -1,37 +1,29 @@
 define ocf::repackage(
     $package     = $title,
-    $recommends  = true,
+    $recommends  = false,
```

Comment:
Yeah, I do agree with that, we don't want to mix behaviors like recommends and backporting, as you said. I'll change it back and add back the `recommends => false` to all those packages.